plugins/qtosgrave/qtosgviewer.cpp

```diff
@@ -2112,21 +2127,34 @@ void QtOSGViewer::_UpdateEnvironment()
     }
 }
 
-void QtOSGViewer::_PostToGUIThread(const boost::function<void()>& fn, bool block)
+size_t QtOSGViewer::_GetQueueSizeForPriority(GUIThreadQueuePriority priority) {
+    size_t queueSize = 0;
+    for (uint8_t i = static_cast<uint8_t>(priority); i < 4; i++) {
+        queueSize += _mapGUIFunctionLists[static_cast<GUIThreadQueuePriority>(i)].size();
```

Comment:
This is doing something different judging from the name. The function name indicates you are getting viewer command number for a specific priority